src/game/server/swarm/asw_marine.cpp

```diff
@@ -4251,6 +4261,31 @@ void CASW_Marine::Event_Killed( const CTakeDamageInfo &info )
 	}
 
 	m_bSlowHeal = false;	// no healing if we're dead!
+
+	CASW_Inhabitable_NPC* pAttackerNPC = dynamic_cast< CASW_Inhabitable_NPC* >( pAttacker );
+	if ( pAttackerNPC && pAttackerNPC->Classify() != CLASS_ASW_MARINE )
```

Comment:
This will prevent marines from turning into zombies on friendly fire kills even if they have the flag set. Is that intended?

Reply:
Hm, i guess not